releasenotes/notes/wsgi-add-streamed-responses-to-request-span-0951a8f8633d9eac.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    wsgi: add span modifiers for request, application, and response spans.
```

Comment:
With the internalization changes we can remove this note and we have no user-impact 😌